twitter/twitter.php

```diff
@@ -791,8 +791,8 @@ function (array $attributes, array $author_contact, $content, $is_quote_share) {
 
 function twitter_addon_admin_post(App $a)
 {
-	$consumerkey    = !empty($_POST['consumerkey'])    ? Strings::escapeTags(trim($_POST['consumerkey']))    : '';
-	$consumersecret = !empty($_POST['consumersecret']) ? Strings::escapeTags(trim($_POST['consumersecret'])) : '';
+	$consumerkey    = trim($_POST['consumerkey'] ?? : '');
```

Comment:
The `:` produces an error 500 and WSOD.

Changing it to...
`$consumerkey    = trim($_POST['consumerkey'] ?? '');`
...solves the issue.

Reply:
Welp. This is only fixing the admin panel. Other pages still don't work.

Reply:
Tumblr has the same issue...

```
worker [CRITICAL]: Uncaught Exception ParseError: "syntax error, unexpected ':'" at /var/www/html/libranet/addon/tumblr/tumblr.php line 78 {"exception":"ParseError: syntax error, unexpected ':' in /var/www/html/libranet/addon/tumblr/tumblr.php:78\nStack trace:\n#0 /var/www/html/libranet/src/Core/Hook.php(193): Friendica\\Core\\Hook::callSingle()\n#1 /var/www/html/libranet/src/Model/Item.php(963): Friendica\\Core\\Hook::callAll()\n#2 /var/www/html/libranet/src/Model/Post/Delayed.php(230): Friendica\\Model\\Item::insert()\n#3 /var/www/html/libranet/src/Worker/DelayedPublish.php(42): Friendica\\Model\\Post\\Delayed::publish()\n#4 [internal function]: Friendica\\Worker\\DelayedPublish::execute()\n#5 /var/www/html/libranet/src/Core/Worker.php(497): call_user_func_array()\n#6 /var/www/html/libranet/src/Core/Worker.php(387): Friendica\\Core\\Worker::execFunction()\n#7 /var/www/html/libranet/src/Core/Worker.php(112): Friendica\\Core\\Worker::execute()\n#8 /var/www/html/libranet/bin/worker.php(86): Friendica\\Core\\Worker::processQueue()\n#9 {main}"} - {"file":null,"line":null,"function":null,"uid":"06f7d6","process_id":3076641}
```

Reply:
Fixing the unexpected `:` in both files solved it.